### PR TITLE
fixes until step 4 PCA pipeline

### PR DESCRIPTION
**Pull request by dianacornejo**
So far I've tested the MWE-AD until step 4 and it runs good.
Still need to test the rest and fix the ggplot output as suggested by @gaow in [here](https://github.com/gaow/lab-wiki/issues/2)
I want to push this changes do they don't accumulate!

### CODE REVIEW

**Comment by gaow** [comment]
Reads good so far @dianacornejo ! looking to your additional pushes.

**Comment by gaow** [comment]
@dianacornejo Also may I ask you to start a new section just to jot down all the actual commands you used? Don't replace my Step 1 to 10 where i put in pseudo commands because i think those help it conceptually cleaner. But an actual section of codes used would help too.

File: GWAS/GWAS_QC.ipynb [hunk 1]
```diff
@@ -276,7 +303,7 @@
     "input: genoFile, group_by=1\n",
     "output: f'{cwd}/cache/{_input:bn}.{name}.filtered.bed'\n",
     "task: trunk_workers = 1, walltime = walltime, mem = mem, cores = numThreads, tags = f'{step_name}_{_output:bn}'\n",
-    "bash: container=container_lmm, expand= \"${ }\", stderr = f'{_output:n}.stderr', stdout = f'{_output:n}.stdout'\n",
+    "bash: container=container_lmm, volumes=[f'{cwd}:{cwd}'],expand= \"${ }\", stderr = f'{_output:n}.stderr', stdout = f'{_output:n}.stdout'\n",
```
[the remark this answers is not in the document]
Comment (dianacornejo) [reply GWAS/GWAS_QC.ipynb:306]: @gaow that volume is necessary if using the docker image.

[the remark this answers is not in the document]
Comment (gaow) [reply GWAS/GWAS_QC.ipynb:306]: @dianacornejo my bad. indeed that is necessary. Please add it back ... i removed it yesterday.

**Pull request merged**